tutorial/arrow-integration-malibu-advanced.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Arrow Integration to Malibu Advanced
+parent: Malibu Advanced Tutorial
```

Comment:
Did `Malibu Advanced Tutorial` exist as parent ? i think you need to create a file and assign it as parent , see how `Malibu Tutorial` has been written as parent .

Reply:
sure will check it